libs/execution_context/ebpf_strings.c

```diff
@@ -5,24 +5,18 @@
 #include <ntstatus.h>
 #include <ntstrsafe.h>
 
-/*
-long
-_ebpf_core_strtol(_In_reads_(str_size) const char* str, size_t str_size, uint64_t flags, _Out_ long* result);
-
-long
-_ebpf_core_strtoul(_In_reads_(str_size) const char* str, size_t str_size, uint64_t flags, _Out_ unsigned long* result);
-*/
-
 // errno_t bpf_strcpy(char *restrict dest, size_t dest_size, const char *restrict src, size_t count);
```

Comment:
This line looks like commented out code.  Either remove it or add English text explaining what this is.